model/states.go

```diff
@@ -292,8 +292,11 @@ func (j *EventBasedSwitchState) UnmarshalJSON(data []byte) error {
 		return err
 	}
 
-	if err := json.Unmarshal(eventBasedSwitch["timeouts"], &j.Timeouts); err != nil {
-		return err
+	eventBaseTimeoutsRawMessage, ok := eventBasedSwitch["timeouts"]
+	if ok {
```

Comment:
I think it is not needed, right?

Reply:
Why not?

Reply:
We must have this because the `timeouts` field is optional. If we don't have it, `checkcarvitals.json` and `eventbasedswitchstate.json` will failed with `unexpected end of JSON input` because we try to unmarshal empty string.